src/tests/ui.md

```diff
@@ -192,7 +192,7 @@ They have several forms, but generally are a comment with the diagnostic level
 to write out the entire message, just make sure to include the important part of
 the message to make it self-documenting.
 
-The error annotation needs to match with the line of the diagnostic. There are
+Most error annotations need to match with the line of the diagnostic. There are
```

Comment:
I'm not sure if we need this change; `~?` actually matches the line **not** having the following message.
Emphasizing "not" would be enough for me if it's unclear.

Reply:
Yeah, `//~?` is used to match diagnostics *without* line information *in the main test file*.